src/include/fcoe-client/complex.rb

```diff
@@ -774,9 +759,7 @@ def HandleInterfacesDialog(id, event)
       nil
     end
 
-    def HandleConfigurationDialog(id, event)
-      event = deep_copy(event)
-      action = Ops.get(event, "ID")
+    def HandleConfigurationDialog(_id, _event)
```

Comment:
what is the reason why this cannot be removed completely?

Reply:
We could, but I just have to stop the refactoring somewhere.

Reply:
you're touching it anyway and dropping unused method makes more sense to me over adding '_' to satisfy a convention ;-)

Reply:
An *empty* method is not an *unused* method. You have missed my point.